src/process-utils.ts

```diff
@@ -0,0 +1,28 @@
+import {exec} from '@actions/exec';
+import {PathLike} from 'fs';
+
+/**
+ * Run actions/exec.exec and return the output from stdout.
+ *
+ * @param cmd command to execute.
```

Comment:
You also need to mention the type of the arguments like [this](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns).

Reply:
Hmm, is this not just for when annotating JavaScript to be able to produce typing files? The page you linked says the following at the top:

> The list below outlines which constructs are currently supported when using JSDoc annotations to provide type information in JavaScript files.

I'm thinking that as this is written in TypeScript, the type is already available in the function header, and so it doesn't need to be duplicated in the docstring.

It does seem like there are different libraries to do this, and [TSdoc looks the most promising to me](https://tsdoc.org/). Looks like I should write it like this, though:

```typescript
/**
 * @param cmd - Command to execute
 */
```

Thoughts?

Reply:
> I'm thinking that as this is written in TypeScript, the type is already available in the function header, and so it doesn't need to be duplicated in the docstring.

I agree with this.

> @param cmd - Command to execute

Yeah, this is better because of the proper segregation of the parameter name and its description.

Reply:
Oh darn, I just merged without fixing this... This is why you shouldn't merge your own PRs xD

I'll invite you to have merge rights so you can merge PRs that you've reviewed.

Reply:
> Oh darn, I just merged without fixing this... This is why you shouldn't merge your own PRs xD

Maybe a separate PR later? You can decide the format of the docstrings on the basis of the documentation generator you use. All of them ideally should have the same convention but you can still check.

> I'll invite you to have merge rights so you can merge PRs that you've reviewed.

Accepted it :)